### ISSUE TITLE
Update H95 TTML and IMSC guidance

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/wcag#2079]
Technique H95 mentions TTML but the link to it is outdated. It includes one polyfill that does not yet claim to support TTML.

Since it was written, the IMSC Profiles of TTML have been published as W3C Recommendations (v1.0.1, v1.1 and v1.2 are all active now), and the [sandflow/imscJS](https://github.com/sandflow/imscJS/) polyfill library has been written to support it - CC @palemieux . The IMSC Profiles are widely adopted in broadcast standards and should be added to the list in H95. There are also pages on MDN that help to explain its usage.

Proposed changes:
1. Update link for TTML to point to https://www.w3.org/TR/ttml/all/
2. Add link to IMSC Profiles at https://www.w3.org/TR/ttml-imsc/all/
3. Add link to imscJS library at https://github.com/sandflow/imscJS/ 
4. Add link to IMSC documentation on MDN at https://developer.mozilla.org/en-US/docs/Related/IMSC

**Comment by nigelmegitt** [comment]
I realised that G87 should probably also be updated to mention IMSC first - the current apparent focus on SMIL is probably not as helpful to most users of the document.

**Comment by detlevhfischer** [comment]
@nigelmegitt 
Hi Nigel, I have added the links you suggestd here https://github.com/w3c/wcag/pull/2150
Hope it is correct.

**Comment by awkawk** [comment]
@detlevhfischer did you mean to remove WebVTT?

**Comment by nigelmegitt** [comment]
Thanks @detlevhfischer much improved, looks good to me. Thank you!

@awkawk Looks like WebVTT is still there.

One question: should we still include Captionator? Looking at https://github.com/cgiffard/Captionator it has not been updated for many years, and has been archived. However https://captionatorjs.com/ is still live. I don't have any contact with the owners, but the motivation for it seems to be primarily as a polyfill for WebVTT, which now has some level of implementation in most browsers.

**Comment by awkawk** [comment]
Sorry, I saw the WebVTT in red on the left side and didn't notice it lower down on the right side in the code changes view. All good.
